### PR TITLE
DCJ-660: Consolidate logger usage

### PR DESCRIPTION
**Pull request by rushtong**
### Addresses
https://broadworkbench.atlassian.net/browse/DCJ-660

### Summary
Minor refactoring of log usage to use `ConsentLogger` where possible.

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by fboulnois — APPROVED** [review]
lgtm 👍

File: src/main/java/org/broadinstitute/consent/http/db/mapper/RowMapperHelper.java [hunk 1]
```diff
@@ -121,8 +118,7 @@ default DataAccessRequestData translate(String darDataString) {
       try {
         data = DataAccessRequestData.fromString(escapedDataString);
       } catch (JsonSyntaxException | NullPointerException e) {
-        String message = "Unable to parse Data Access Request; error: " + e.getMessage();
-        log.error(message);
+        logDebug("Unable to parse Data Access Request; error: %s".formatted(e.getMessage()));
```
Comment (rjohanek) [inline src/main/java/org/broadinstitute/consent/http/db/mapper/RowMapperHelper.java:121]: should this be logException or do we want to change it to debug?

File: src/main/java/org/broadinstitute/consent/http/mail/SendGridAPI.java [hunk 2]
```diff
@@ -94,7 +93,7 @@ private boolean findUserEmailPreference(Mail message) {
       if (!tos.isEmpty()) {
         User user = userDAO.findUserByEmail(tos.get(0).getEmail());
         if (Objects.isNull(user)) {
-          logger.error("Unknown user: " + tos.get(0).getEmail());
+          logWarn("Unknown user: %s".formatted(tos.get(0).getEmail()));
```
Comment (rjohanek) [inline src/main/java/org/broadinstitute/consent/http/mail/SendGridAPI.java:96]: should this be logException or do we want to change it to warn?

**Review by rjohanek — APPROVED** [review]
couple questions, but looks good

File: src/main/java/org/broadinstitute/consent/http/mail/SendGridAPI.java (hunk 2 above)
Comment (rushtong) [reply src/main/java/org/broadinstitute/consent/http/mail/SendGridAPI.java:96]: Ah yes, this was an intentional change - I should have commented on that. `logException` triggers a sentry alert which is overkill for this kind of problem.

File: src/main/java/org/broadinstitute/consent/http/db/mapper/RowMapperHelper.java (hunk 1 above)
Comment (rushtong) [reply src/main/java/org/broadinstitute/consent/http/db/mapper/RowMapperHelper.java:121]: This is intentional too - we don't want to spam prod sentry with this kind of error.

**Review by s-rubenstein — APPROVED** [review]
LGTM

**Pull request merged**